packages/web-react/src/components/Modal/demo/ModalScrollingLongContent.tsx

```diff
@@ -16,12 +56,42 @@ const ModalScrollingLongContent = () => {
   const handleModalScrollingInsideClose = () => setModalScrollingInsideOpen(false);
   const handleModalCustomHeightClose = () => setModalCustomHeightOpen(false);
 
+  const handleMaxHeightChange = (event: ChangeEvent<HTMLInputElement>, breakpoint: BreakpointToken) => {
+    setHeightValue((prevState) => ({
```

Comment:
You can create a function and pass it: 
```suggestion
    setHeightValue((prevState) => setVerticalDimensionValueForProp(prevState, 'height'))
```

```js
const setVerticalDimensionValueForProp = (prevState, propName) => {
  return ({
  ...prevState,
     [propName]: {
        ...prevState[propName],
        [breakpoint]: Number(event.target.value),
      },
  })
}
```